src/main/java/org/gridsuite/explore/server/services/DirectoryService.java

```diff
@@ -91,9 +91,14 @@ public ElementAttributes getElementInfos(UUID elementUuid) {
         return restTemplate.exchange(directoryServerBaseUri + path, HttpMethod.GET, null, ElementAttributes.class).getBody();
     }
 
-    private List<ElementAttributes> getElementsInfos(List<UUID> elementsUuids) {
+    private List<ElementAttributes> getElementsInfos(List<UUID> elementsUuids, List<String> elementTypes) {
         var ids = elementsUuids.stream().map(UUID::toString).collect(Collectors.joining(","));
         String path = UriComponentsBuilder.fromPath(ELEMENTS_SERVER_ROOT_PATH).toUriString() + "?ids=" + ids;
+
+        if (elementTypes != null) {
```

Comment:
! CollectionUtils.isEmpty(elementTypes) ?
to avoid pb if elementTypes is not null but empty ...

Reply:
done